src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -288,15 +288,15 @@ private MetadataResolver createMetadataResolver(final Settings settings, final P
 
         final String idpMetadataUrl = settings.get("idp.metadata_url");
         final String idpMetadataFile = settings.get("idp.metadata_file");
-        final String idpMetadataBody = settings.get("idp.metadata_body");
+        final String idpMetadataBody = settings.get("idp.metadata_content");
```

Comment:
Please introduce `public static final String`.

Reply:
These values are being set in a private method. This will need us to define it at class level and initialize the values in the constructor. Any reason for preferring this?

Reply:
It is a standard practice to define constant string as a class level variable.